pkgs/by-name/fl/flashlabel-yxwl/package.nix

```
@@ -0,0 +1,83 @@
+{
+  autoPatchelfHook,
+  cups,
+  lib,
+  requireFile,
+  stdenv,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "flashlabel-yxwl";
+  version = "1.2.1";
+
+  # The source URL currently redirects through “pCloud”, a file storage service
+  # that resists direct downloads.
+  src = requireFile {
+    name = "A4_Linux_Driver_Ver${version}.run";
+    url = meta.downloadPage;
```

I'd avoid this personally and leave the full url instead as I don't anyone else doing so in nixpkgs.
```suggestion
    url = "https://flashlabel.net/YXWL-A4driver-linux";
```